### DISCUSSION TITLE
Nextjs preview does not work with prebuilt vecel cli deploy

### DISCUSSION THREAD

**Discussion by zhex900** [vercel/vercel#8185] [Help] [answered]
Hi,

My Nextjs project's preview mode goes to 505 page when deployed using prebuilt artifact via vercel cli deploy. 

But the preview mode works when deployed using vercel pipeline. 

Why is that? Is there any special config?

**Comment by leerob** [comment, marked as answer]
Hey there! To confirm, are you using `vercel build` followed up `vercel deploy --prebuilt` when you say "prebuilt artifact"?

Can you share the source code or is there a minimal reproducible example?

**Reply by zhex900** [reply]
Hey,

I think I know the problem now. I built the source on my mac. Now, I have fixed it by running the install and build in a Linux container.

All good now.
